wordpress-mobile/WordPress-Android
# [JP Social/Pre-publishing] Add JP Social placeholder item to bottom sheet

Part of #18694 

This adds the `PrepublishingHomeSocialItem` composable inside the pre-publishing bottom sheet (using fake data currently) if the `JetpackSocialFeatureConfig` feature-in-development is turned on in Debug Settings.

This integrates the View-based and Compose-based parts of this UI, even though it still doesn't retrieve real data (which will be done in a different PR).

To test:
1. Open the Jetpack App
2. Go to the Debug Settings
3. Go to `Features in Development`
4. Enable `JetpackSocialFeatureConfig`
5. Open/Create a post
6. Hit `Publish`/`Update`
7. **Verify** the JP Social item is showing (with fake data)
    1. title: "Sharing to 2 of 3 accounts"
    2. description: "27/30 shares remaining"
    3. icons: disabled facebook icon and enabled tumblr icon
8. Disable `JetpackSocialFeatureConfig`
9. Open/Create a post
10. Hit `Publish`/`Update`
11. **Verify** the JP Social item is NOT showing

## Regression Notes
1. Potential unintended areas of impact
Behavior/wrong UI in the pre-publishing sheet

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
WIP

PR submission checklist:


UI Changes testing checklist:
